search/conf.py

```diff
@@ -0,0 +1,29 @@
+"""search application configuration"""
+from appconf import AppConf
+from django.conf import settings
+
+
+class SearchAppConf(AppConf):
+    """Configuration class for a configurable CAS authentication system"""
+
+    # ENABLED = False
```

Comment:
The configuration variable, of which you haven't added one yet, but commented out one: http://django-appconf.readthedocs.org/en/latest/ for docs.  That may help you understand, if it doesn't feel free to hit me up on chat or something.